src/Map.cpp

```diff
@@ -65,22 +65,22 @@ void cMap::Tick()
 
 
 
-void cMap::UpdateRadius(int a_PixelX, int a_PixelZ, unsigned int a_Radius)
+void cMap::UpdateRadius(unsigned int a_PixelX, unsigned int a_PixelZ, unsigned int a_Radius)
 {
-	int PixelRadius = a_Radius / GetPixelWidth();
+	unsigned int PixelRadius = a_Radius / GetPixelWidth();
 
-	unsigned int StartX = Clamp(a_PixelX - PixelRadius, 0, (int)m_Width);
-	unsigned int StartZ = Clamp(a_PixelZ - PixelRadius, 0, (int)m_Height);
+	unsigned int StartX = Clamp(a_PixelX - PixelRadius, static_cast<unsigned int>(0), m_Width);
+	unsigned int StartZ = Clamp(a_PixelZ - PixelRadius, static_cast<unsigned int>(0), m_Height);
```

Comment:
Using `unsigned int` gives us an implicit check for 0, which we would have to add to multiple places throughout the map code to make sure everything would work fine with `int`. Also, a_PixelX and PixelRadius are both unsigned.

What really needs to be done is that I need to rework cMap::UpdateRadius(cPlayer & a_Player, unsigned int a_Radius) so that the values being passed in are relative to the starting point of the map.